lib/index.js

```diff
@@ -47,17 +49,15 @@ const highlighters = [
   }
 ]
 
-function highlight (sqlString, options) {
-  options = Object.assign({}, DEFAULT_OPTIONS, options)
-
+function getSegments (sqlString) {
   const matches = []
 
   for (const hl of highlighters) {
     let match
 
     // This is probably the one time when an assignment inside a condition makes sense
     // eslint-disable-next-line no-cond-assign
-    while (match = hl.regex.exec(sqlString)) {
+    while ((match = hl.regex.exec(sqlString))) {
```

Comment:
What's the reason for these extra parentheses?

Reply:
this came through prettier. To be honest I can't explain the sense either and would take it away again.